eZ/Bundle/EzPublishCoreBundle/Resources/config/templating.yml

```
@@ -184,46 +184,22 @@ services:
 
     eZ\Publish\Core\MVC\Symfony\Templating\RenderContentStrategy:
         arguments:
-            $defaultMethod: 'inline'
-            $renderMethods: !tagged_iterator ibexa.platform.render.method
+            $fragmentRenderers: !tagged_iterator kernel.fragment_renderer
+            $defaultRenderer: 'direct'
             $siteAccess: '@ezpublish.siteaccess'
             $requestStack: '@request_stack'
         tags:
             - { name: ibexa.platform.render.strategy }
 
     eZ\Publish\Core\MVC\Symfony\Templating\RenderLocationStrategy:
         arguments:
-            $defaultMethod: 'inline'
-            $renderMethods: !tagged_iterator ibexa.platform.render.method
+            $fragmentRenderers: !tagged_iterator kernel.fragment_renderer
+            $defaultRenderer: 'direct'
```

We can use class constant instead (since we already have it declared). Double check if my syntax is correct.

```suggestion
            $defaultRenderer: !php/const eZ\Bundle\EzPublishCoreBundle\Fragment\DirectFragmentRenderer::NAME
```